elements/element-stuff-base.html

```diff
@@ -37,11 +38,17 @@
             return;
           }
 
+          // FIXME : A bit silly but useful for custom-element cases
+          let _value = target.value;
```

Comment:
Instead of this, I think there should be a special class name on the colour/background colour inputs, so that we can write an `if` statement similar to the one above.

Reply:
or, `let value = target.value || target.color`, but i'm worried that if the `value` is false or undefined it would fallback to the `color` incorrectly

Reply:
The second is a more elegant solution, but I'll go with the if statement to avoid possible issue here